docs/fuzz-settings.md

```diff
@@ -1,137 +1,356 @@
 
-## Corpus
+Default: depends on the fuzz test runner (CLI/Jest)
+
+Set the corpus directories or the individual corpus files to use for fuzzing
+(mixing directories and files is not allowed).
 
 Jazzer.js generates meaningful inputs to a fuzz target based on coverage and
 comparison feedback. If a new input can reach new code paths, it is saved in a
-dedicated directory, called corpus, and used for further mutations to the guide
-the fuzzer during the following iterations.
-
-Also, existing inputs in the corpus directory, called seeds, are executed on
-startup, so that new fuzzing runs can start off where previous ones stopped.
+dedicated directory, called _corpus directory_, and used for further mutations
+to the guide the fuzzer during the following iterations. Existing inputs in the
+corpus directory, called _seeds_, are executed on startup, so that new fuzzing
+runs can start off where previous ones stopped.
 
 One or more corpus directories can be specified as the last entry/entries in the
-CLI parameter list, as described in the `--help` command. The first corpus
-directory will be used to save interesting new inputs, whereas seeds from all
-directories are executed during startup.
+parameter list. The first corpus directory will be used to save interesting new
+inputs, whereas seeds from all directories are executed during startup.
+
+If the corpus points to files instead of directories, Jazzer.js will run the
+fuzz target with each file as input once and exit. This can be used to manually
+reproduce previously found issues.
 
-**Example invocation:**
+**CLI:** Default: `[]`
+
+This is a positional argument that should be specified without a flag as the
+last argument. Here is an example how to specify two corpus directories
+`corpus_dir` and `only_on_startup_corpus` in CLI mode.
 
 ```shell
-npx jazzer target corpus_dir other_corpus
+npx jazzer fuzz-file corpus_dir only_on_startup_corpus
 ```
 
-## Reproducing errors
+It is also possible to provide inputs directly. In that case, Jazzer.js will run
+them once and exit. To run the fuzz target with inputs `crash-abcdef0123456789`
+and `my-test.txt` in CLI can be done as follows:
 
-Once Jazzer.js finds a problematic input, it stores it in the current working
-directory using a problem prefix like `crash-`, `mem-`, `timeout-` or the like.
+```shell
+npx jazzer fuzz-file crash-abcdef0123456789 my-test.txt
+```
 
-This input can then be used to reproduce the issue by specifying it as last
-parameter in the CLI call:
+If no corpus directory is provided, Jazzer.js will start fuzzing from scratch
+and will not save any inputs other than crashes on exit.
 
-```shell
-npx jazzer target crash-abcdef0123456789
+**Jest:** In Jest runner mode, Jazzer.js automatically chooses the corpus
+directories based on the name of the fuzz test and the
+[mode](#mode--fuzzingregression) ("fuzzing" or "regression").
+
+In _regression mode_, the main corpus directory is the directory where the test
```

Comment:
Why is the distinction between fuzzing and regression mode necessary? Seeds should be placed in the `<test-file-name>/<test-blocks>` directory in `<root>` in both cases.

Reply:
I updated the description to be primarily about generated/regression corpus and less about the mode. Though I prefer to say "regression corpus" instead of the "seed corpus" (which tells nothing about the purpose of the corpus) :smile: